content/blog/elao/amelie-integratrice-depuis-5-ans.md

```diff
@@ -0,0 +1,93 @@
+---
+title: 'Amélie, intégratrice chez Elao depuis 5 ans'
+date: '2022-02-22' # Au format YYYY-MM-DD
+lastModified: ~ # À utiliser pour indiquer explicitement qu'un article à été mis à jour
+description: 'petite phrase sympa sur la personne'
+authors: [elao, adefrance] # multiple acceptés
+#tableOfContent: true # `true` pour activer ou `3` pour lister les titres sur 3 niveaux.
+tags: [elao, team]
+thumbnail: images/posts/thumbnails/trame-itw.jpg
+#banner: images/posts/headers/trame-itw.jpg # Uniquement si différent de la minitature (thumbnail)
+#credit: { name: 'Thomas Jarrand', url: 'https://unsplash.com/@tom32i' } # Pour créditer la photo utilisée en miniature
+#tweetId: ~ # Ajouter l'id du Tweet après publication.
+---
+
+<!-- INTRO -->
+Aujourd'hui, on vous présente Amélie, inté chez Elao depuis 5 ans.
+
+## Ton parcours
+
+## C'est quoi que tu préfères dans le quotidien de ton métier ?
+
+Intégrer une maquette bien pensée, qui va s'adapter comme il faut à un vrai projet avec des vraies data, où tout tombe comme ça devrait 📐
+
+## Et ce que tu laisses volontiers aux autres ?
+
+La confection de ladite maquette.
+
+### Qu'est-ce que tu fais quand tu ne travailles pas ?
+
+Plein de choses ! J'ai un chat donc déjà je ramasse beaucoup d'objets cassés. Autrement je couds, je tricote (actuellement un gros gilet qui ne sera jamais fini au rythme actuel) ou je binge des séries. J'ai une très chouette île sur Animal Crossing.
+
+## L'interview :fast-parrot: 
+
+### Tes 3 emojis préf du Slack
+
+✨ pour tout ce qui me satisfait
+
+🔥 pour ce qui me satisfait moins
+
+:ultra_fast_parrot: pour les émotions intenses
+
+### Les 3 trucs qui traînent toujours autour de ton bureau
+
+Mais du coup :
+- Une boule à thé
+- Un thé pas terminé parce que les petits bouts de feuille sont tombés au fond
+- Du papier et un stylo pour faire des schémas
+
+### Le projet sur lequel tu aimerais vraiment travailler
+
+### Plutôt Slack ou call ?
+
+Slack sans hésiter. Parce que contrairement à un call, on peut dessiner ce qu'on pense.
+
+Par exemple voici mon gâteau d'anniversaire de 2021 :
+```
+
+                            🔥 bougie bougie 🔥
+
+              | meringue italienne, framboises, myrtilles |
+               -------------------------------------------
+              |        truc fruité à la framboise         |
+truc vanillé  |                 génoise                   |  truc vanillé
+              |      sorte de gelée aux fruits rouges     |
+              |                 génoise                   |
+               -------------------------------------------
+
+```
+
+Et mon appartement :
+
+```
+-----------------------------------
```

Comment:
And it was this moment Jackson knew, he fucked up:

antikor medium, la font qu'on a pour les balises code n'est pas monospaced 😁

Reply:
Ouais j'suis deg. Je vais donc aller faire des screenshots sur Slack et mon effet est gâché 🤷‍♀️

Reply:
On en profiterait pas pour changer de font du coup ?

Reply:
Tant pis pour cette fois mais oui j'ai fait une issue